scss/_settings_breakpoints.scss

```diff
@@ -1,6 +1,6 @@
 // Global breakpoints
 $breakpoint-x-small: 460px !default;
-$breakpoint-small: 620px !default;
+$breakpoint-small: 800px !default;
```

Comment:
@lyubomir-popov Why is small breakpoint changing?

It makes the range for "tablet" only between 800 and 1036px, it's just 200 pixels. Seems quite small range to maintain.

Now everything below 800 will end up in "mobile" view.